src/lib/DatePickerComponent.android.js

```diff
@@ -159,69 +158,69 @@ export class DatePickerComponent extends React.Component {
       : "";
 
     const timeValue = this.state.date || new Date();
-    const valueTestId = this.state.date
-      ? `Value[${this.state.date.getTime()}]`
-      : "Value[unknown]";
+
+    const valueTestId =
+      `Value/` + this.state.date
+        ? `Value/${this.state.date?.getTime()}`
```

Comment:
`'Value/' + this.state.date` as a string of length > 0 will always evaluate to truthy, so the "Unknown" clause will never be.  ditto the ones below